HunnitLog/HunnitLog/View/Home/ScorePad.swift

```diff
@@ -0,0 +1,126 @@
+            GridItem(.fixed(Constants.feelingsDiameter), spacing:Constants.feelingSpacing)
+        ]
+    }
+    
+    // MARK: - fake action functions: 월 숫자 보정
+    func goPrev() {
+        self.showingMonth = showingMonth == 1 ? 12 : (showingMonth - 1) % 12
+        self.dateComponents.month = showingMonth
+    }
+    func goNext() {
+        self.showingMonth = showingMonth == 11 ? 12 : (showingMonth + 1) % 12
+        self.dateComponents.month = showingMonth
+    }
+    
+    // MARK: - 더미 데이터 구조체
+    struct dailyFeeling: Hashable {
+        var day: Int
+        var feeling: Feeling
+    }
+    
+    // MARK: - 더미 데이터
+    private var feelings = [
+        dailyFeeling(day: 1, feeling: Feeling.good),
+        dailyFeeling(day: 2, feeling: Feeling.notBad),
+        dailyFeeling(day: 3, feeling: Feeling.excellent),
+        dailyFeeling(day: 4, feeling: Feeling.bad),
+        dailyFeeling(day: 5, feeling: Feeling.soso)
+    ]
+    
+    var body: some View {
+        let range = calendar.range(of: .day, in: .month, for: calendar.date(from:dateComponents)!)
+        let numDays = range!.count
+        
+        ZStack {
+            Rectangle()
+                .cornerRadius(Constants.backgroundRadius)
+                .foregroundColor(CustomColor.bgColor)
+            VStack {
+                HStack {
+                    Button(action: goPrev, label: {
+                        // TODO: - 아이콘 색 분기처리
+                        Image(Constants.leftImageName)
+                            .foregroundColor(CustomColor.black)
+                    })
+                    Spacer()
+                    Text("\(String(currentYear))년 \(showingMonth)월")
+                        .font(Constants.titleFont)
+                    Spacer()
+                    Button(action: goNext, label: {
+                        Image(Constants.rightImageName)
+                            .foregroundColor(CustomColor.black)
+                    })
+                    
+                }
+                LazyVGrid(columns: columns, spacing:5, content: {
+                    ForEach(0..<numDays) { f in
+                        if f < feelings.count {
+                            feelings[f].feeling.icon
+                                .resizable()
+                                .frame(width:Constants.feelingsDiameter, height:Constants.feelingsDiameter)
+                        } else {
+                            ZStack {
+                                Circle()
+                                    .frame(width:Constants.feelingsDiameter, height:Constants.feelingsDiameter)
+                                    .foregroundColor(CustomColor.lightGray)
+                                Text("\(f + 1)")
+                                    .font(Constants.numberFont)
+                                    .foregroundColor(CustomColor.coolGray)
+                            }
+                        }
+                    }
+                })
```

Comment:
뷰모델이 어떻게 올진 모르겠지만,, 만약 작성해주신 것과 동일하게 온다면 저는 이런식으로 day에 맞는 feeling을 설정해 줄 듯합니다~
```suggestion
var feelingSort = feelings.sorted(by: {$0.day < $1.day})
LazyVGrid(columns: columns, spacing:5, content: {
    ForEach(1...numDays, id: \.self) { day in
        if (feelingSort.first?.day == day) {
            feelingSort.removeFirst().feeling.icon
                .resizable()
                .frame(width:Constants.feelingsDiameter, height:Constants.feelingsDiameter)
        } else {
            ZStack {
                Circle()
                    .frame(width:Constants.feelingsDiameter, height:Constants.feelingsDiameter)
                    .foregroundColor(CustomColor.lightGray)
                Text("\(day)")
                    .font(Constants.numberFont)
                    .foregroundColor(CustomColor.coolGray)
            }
        }
    }
})
```

Reply:
오 그러고보니 day값들이 순서대로 들어오는 게 아니네요..! 
기록을 안 한 날이 있을 수 있으니까... 감사합니다 수정했습니다!!